src/divbuf.rs

```diff
@@ -388,6 +389,44 @@ impl From<Vec<u8>> for DivBufShared {
     }
 }
 
+impl TryFrom<DivBufShared> for Vec<u8> {
+    type Error = DivBufShared;
+
+    /// Attempt to extract the owned storage from a DivBufShared.
+    ///
+    /// This will fail if there are any other living DivBufs for this same
+    /// DivBufShared, in which case the DivBufShared will be returned
+    /// unmodified.
```

Comment:
```suggestion
    /// This will fail if there are any other living references to this same
    /// `DivBufShared` (`DivBuf`s, `DivBufMut`s, etc), in which case the
    /// `DivBufShared` will be returned unmodified.
```

Reply:
Done.